src/TMxpMusicTagHandler.cpp

```diff
@@ -29,7 +29,9 @@ TMxpTagHandlerResult TMxpMusicTagHandler::handleStartTag(TMxpContext& ctx, TMxpC
 
     mediaData.setMediaProtocol(TMediaData::MediaProtocolMSP);
     mediaData.setMediaType(TMediaData::MediaTypeMusic);
-    mediaData.setMediaFileName(tag->getAttributeValue("FName"));
+    if (tag->hasAttribute("FName")) {
+        mediaData.setMediaFileName(tag->getAttributeValue("FName"));
+    }
```

Comment:
If https://github.com/Mudlet/Mudlet/pull/4577/files#r551074385 is the case then should there not be a further change here:
```suggestion
    } else {
        mediaData.setMediaFileName(QStringLiteral("off"));
    }
```
?